### PR TITLE
feat(MI-86): updating uploads/creatives API with personalized QR codes in campaigns

### PR DESCRIPTION
**Pull request by amaan-lob**
Fixes MI-86
Functionality already exists, just updating copy to clarify how to use personalized QR codes per recipient in a campaign.

## Checklist

- [x] Up to date with `main`
- [x] All the tests are passing
  - [x] Delete all resources created in tests
- [x] Prettier
- [x] Spectral Lint
- [x] `npm run bundle` outputs nothing suspect
- [x] `npm run postman` outputs nothing suspect

## Changes

- clarify how to specify unique redirect URLs per recipient when using QR codes in campaigns
- update request samples to include qr_code_redirect_url

### CODE REVIEW

File: shared/models/qr_code_campaigns.yml [hunk 1]
```diff
@@ -28,11 +28,12 @@ properties:
...
-      To redirect to a single URL for the whole campaign, add a `redirect_url` in the request body along with the url as string.
-      To redirect to a custom URL for each recipient, do not provide any value for `redirect_url`. Instead, create an extra column in the [audience file](https://help.lob.com/print-and-mail/building-a-mail-strategy/campaign-or-triggered-sends/campaign-audience-guide) with a unique link against each address row and while creating an upload, map `qr_code_redirect_url` to this column.
-      If the QR code section is used but a redirection url is not provided or mapped while creating an upload, then there might be failures in creating individual mail pieces.
+    description: |
+      Redirect all mail recipients to either a single URL or specify a unique URL for each recipient.
+      To redirect all mail recipients in a campaign to the same URL, assign the desired URL to this field.
+      To redirect each mail recipient to a unique URL, leave this field empty and specify the redirect link for each row in the audience file. Then, when creating a new upload, map this column to the `qr_code_redirect_url` key in the `mergeVariableColumnMapping` property.
```
Comment (kevinpjones) [inline shared/models/qr_code_campaigns.yml:34]: Probably want to retain the help center link on audience file.

**Review by kevinpjones — APPROVED** [review]

**Review by juanfriss — APPROVED** [review]
you will need to upgrade package.json too

**Comment by amaan-lob** [comment]
> you will need to upgrade package.json too

Hey @juanfriss, do we need to include output files from the `npm run bundle` and `npm run postman` commands since I see they are executed in the Deploy GHA?

**Comment by juanfriss** [comment]
I think you don't as "The bundled version is generated as part of CI/CD"

**Comment by kevinpjones** [comment]
I think you'll need `npm run bump-version`, as it updates version in package.json as well as the spec yaml

**Pull request merged**